docs/configure/invite-collaborate.md

```diff
@@ -0,0 +1,56 @@
+<!--
+title: "Invite your team and collaborate"
+description: "Invite your SRE, DevOPs, or ITOps teams to Netdata Cloud to give everyone insights into your infrastructure from a single pane of glass."
+custom_edit_url: https://github.com/netdata/netdata/edit/master/docs/configure/invite-collaborate.md
+-->
+
+# Invite your team and collaborate
+
+Netdata is designed to make an infrastructure's real-time metrics available and actionable to all organization members.
+By inviting others, you can better synchronize with your team or colleagues to understand your infrastructure's
+heartbeat. When something goes wrong, you'll be ready to collaboratively troubleshoot complex performance problems from
+a single pane of glass.
+
+## Invite new members
+
+Invite new users by clicking on your Space's name in the top navigation, and then **Invite more users**, to open the
+invitation pane.
+
+![Opening and navigating the invitation
+panel](https://user-images.githubusercontent.com/1153921/92025596-a618e680-ed14-11ea-9c1f-a61fdcb8aa4e.png)
+
+Enter their email address and name. They can change this name once they accept your invitation.
+
+Choose which War Rooms you want to add this user to, then click the plus **+** button to add the invitation to the
```

Comment:
Admins manage user permissions and have control over who can access specific Spaces and War Rooms.